CONTRIBUTING.md

```diff
@@ -90,16 +90,110 @@ For example, the file at `/docs/quickstarts/setup-clerk.mdx` can be found at htt
 
 The navigation element rendered on https://clerk.com/docs is powered by the manifest file at [`/docs/manifest.json`](./docs/manifest.json). Making changes to this data structure will update the rendered navigation.
 
-#### Navigation constructs
+[Manifest JSON schema →](./docs/manifest.schema.json)
+
+<details>
+<summary>Equivalent TypeScript types and descriptions</summary>
+
+```typescript
+export type Nav = Array<NavGroup>
+
+/**
+ * Nav groups are separated by horizontal rules
+ */
+type NavGroup = Array<NavItem>
+
+/**
+ * A nav item is either a link, or a sub-list with nested `items`
+ */
+type NavItem = LinkItem | SubNavItem
+
+/**
+ * A link to an internal or external page
+ */
+type LinkItem = {
+  /**
+   * The visible item text. May contain backticks (`) to render `<code>`
+   * 
+   * @example 'Next.js Quickstart'
+   * @example '`<SignIn>` and `<SignUp>`'
+   */
+  title: string
+  /**
+   * The item link. Internal links should be relative
+   * 
+   * @example '/docs/quickstarts/nextjs'
+   * @example 'https://example.com'
+   */
+  href: string
+  /**
+   * Muted text to display next to the item text
+   * 
+   * @example 'Community'
+   * @example 'Beta'
+   */
+  tag?: string
+  /**
+   * Icon to display next to the item text
+   * 
+   * @example 'globe'
+   * @see [Available icons]{@link https://github.com/clerk/clerk/blob/main/src/app/(website)/docs/icons.tsx}
+   */
+  icon?: string
+  /**
+   * Whether to enable text wrapping for the item text
+   * 
+   * @default true
+   */
+  wrap?: boolean
+}
+type SubNavItem = {
+  /**
+   * The visible item text. May contain backticks (`) to render `<code>`
+   * 
+   * @example 'Next.js Quickstart'
+   * @example '`<SignIn>` and `<SignUp>`'
+   */
+  title: string
+  /**
+   * The nested sub-items
+   */
+  items: Nav
+  /**
+   * Muted text to display next to the item text
+   * 
+   * @example 'Community'
+   * @example 'Beta'
+   */
+  tag?: string
+  /**
+   * Icon to display next to the item text
+   * 
+   * @example 'globe'
+   * @see [Available icons]{@link https://github.com/clerk/clerk/blob/main/src/app/(website)/docs/icons.tsx}
+   */
+  icon?: string
+  /**
+   * Whether to enable text wrapping for the item text
+   * 
+   * @default true
+   */
+  wrap?: boolean
+  /**
+   * Whether to collapse the sub-nav
+   * 
+   * @default false
+   */
+  collapse?: boolean
```

Comment:
Feels like this should be true by default since in most cases (I assume) we want these collapsible lists to start collapsed. cc @kylemac @alexisintech

Reply:
@S3Prototype i think i prefer the defaulting that @bradlc has set up for now. I was using it extensively when i was playing with the `multi-manifest` stuff and it was feeling intuitive

Reply:
Sounds good to me!

Reply:
quick chat over slack. not a blocker for @S3Prototype so we're gonna merge and observe over time ❤️